### PR TITLE
Automatic port of *OSPlatformAttribute docs

### PR DESCRIPTION
**Pull request by carlossanlop**
Ported with [DocsPortingTool](https://github.com/carlossanlop/DocsPortingTool).

### CODE REVIEW

**Review by adamsitnik — APPROVED** [review]
LGTM, thank you @carlossanlop !

File: xml/System.Runtime.Versioning/UnsupportedOSPlatformAttribute.xml [hunk 1]
```diff
@@ -40,8 +48,8 @@
         <Parameter Name="platformName" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="platformName">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="platformName">The unsupported OS platform name.</param>
```
Comment (buyaa-n) [inline xml/System.Runtime.Versioning/UnsupportedOSPlatformAttribute.xml:51]:
```suggestion
        <param name="platformName">The unsupported OS platform name, optionally including a version</param>
```

File: xml/System.Runtime.Versioning/SupportedOSPlatformAttribute.xml [hunk 2]
```diff
@@ -20,8 +20,17 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Records the operating system (and minimum version) that supports an API. Multiple attributes can be applied to indicate support on multiple operating systems.</summary>
```
Comment (buyaa-n) [inline xml/System.Runtime.Versioning/SupportedOSPlatformAttribute.xml:23]:
```suggestion
    <summary>Indicates that an API is supported for a specified platform or operating system. If a version is specified, the API cannot be called from an earlier version. Multiple attributes can be applied to indicate support on multiple operating systems.</summary>
```

File: xml/System.Runtime.Versioning/SupportedOSPlatformAttribute.xml [hunk 3]
```diff
@@ -40,8 +49,8 @@
         <Parameter Name="platformName" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="platformName">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="platformName">The supported OS platform name.</param>
```
Comment (buyaa-n) [inline xml/System.Runtime.Versioning/SupportedOSPlatformAttribute.xml:52]:
```suggestion
        <param name="platformName">The supported OS platform name, optionally including a version</param>
```

File: xml/System.Runtime.Versioning/UnsupportedOSPlatformAttribute.xml [hunk 4]
```diff
@@ -20,8 +20,16 @@
...
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Marks APIs that were removed in a given operating system version.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Primarily used by OS bindings to indicate APIs that are only available in earlier versions.
```
Comment (buyaa-n) [inline xml/System.Runtime.Versioning/UnsupportedOSPlatformAttribute.xml:29]:
```suggestion
Version less attribute primarily used to indicate the API is unsupported for the platform. If a version is specified, the API is only available in earlier versions.
```

File: xml/System.Runtime.Versioning/UnsupportedOSPlatformAttribute.xml [hunk 5]
```diff
@@ -20,8 +20,16 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Marks APIs that were removed in a given operating system version.</summary>
```
Comment (buyaa-n) [inline xml/System.Runtime.Versioning/UnsupportedOSPlatformAttribute.xml:23]:
```suggestion
    <summary>Marks APIs that were removed or are unsupported in a given operating system version.</summary>
```

**Review by buyaa-n — APPROVED** [review]
Added some modifications, thanks for adding this!

File: xml/System.Runtime.Versioning/SupportedOSPlatformAttribute.xml [hunk 6]
```diff
@@ -40,8 +49,8 @@
         <Parameter Name="platformName" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="platformName">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="platformName">The supported OS platform name, optionally including a version.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Runtime.SupportedOSPlatformAttribute" /> attribute class for the specified supported OS platform.</summary>
```
Comment (carlossanlop) [inline xml/System.Runtime.Versioning/SupportedOSPlatformAttribute.xml:53]:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Runtime.Versioning.SupportedOSPlatformAttribute" /> attribute class for the specified supported OS platform.</summary>
```

**Pull request merged**